pom.xml

```diff
@@ -254,7 +254,7 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>${argLine} -Djava.awt.headless=true -Dfile.encoding=UTF-8</argLine>
+          <argLine>${argLine} -Djava.awt.headless=true -Dfile.encoding=UTF-8 -Duser.language=en -Duser.region=US</argLine>
```

Comment:
I don't think that hard-coding language and region is a good idea. Our tests should not depend on that.

Reply:
@t-izbassar On the contrary, putting this encoding and region info ensures that our tests work as intended. We are heaving a lot of problems in other projects due to language issues (mostly number formatting, some locales use "," as decimal point, some use ".") and these setting these locale info on the project assures that we will always use the correct one despite the user machine configuration.

Reply:
@paulodamaso I think the tests that require the language should set it explicitly in tests so that they are visible. But putting global hard-coded value is a mistake.